notebooks/data-sources/gcsweb-ci/build-logs/model_seldon.ipynb

```diff
@@ -0,0 +1,562 @@
+{
```

Comment:
<p>can we get the credentials from the .env by default?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/371/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>This should set the variables with <code>os.getenv()</code>  as ideally these are being stored as envs through the Kube secret. and then if not, via .env file. which would require a the use of <code>dotenv</code>   above. As well as an .env file in this dir specific to this task, right? </p>

Reply:
++ added os.getenv()